services/weightConversions.js

```diff
@@ -83,7 +83,7 @@ const convertToGrams = (weight, weightType) => {
     if (weightType === 'tablespoon'){
         return tbspToGrams(weight);
     }
-    if (weightType === 'dessertspoon'){
+    if (weightType === 'dessert spoon'){
```

Comment:
any specific reason you wanted to add a space here?

Reply:
In the front end, we currently have it as 'Dessert Spoon'. 
In the backend we lowercase it but it came as an error without the space